### PR TITLE
Add compat data for MIDIInput

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/MIDIInput

### CODE REVIEW

File: api/MIDIInput.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MIDIInput",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "43"
+          },
+          "chrome_android": {
+            "version_added": false
```
Comment (jpmedley) [inline api/MIDIInput.json]: Both versions of Android should say `"43"`.

File: api/MIDIInput.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/MIDIInput.json]: Both versions of Opera should say 30.

File: api/MIDIInput.json (hunk 1 above)
Comment (maboa) [reply api/MIDIInput.json]: 👍

File: api/MIDIInput.json (hunk 2 above)
Comment (maboa) [reply api/MIDIInput.json]: 👍

**Comment by maboa** [comment]
Updated :)

**Pull request merged**